theme/templates/base.html

```diff
@@ -44,7 +44,7 @@
 <script src='{% static 'js/jquery-ui.js' %}'></script>
 <script src="{% static "js/bootstrap.3.0.2.min.js" %}"></script>
 <script src="{% static "js/bootstrap-extras.js" %}"></script>
-{% block extra_js %}{% endblock %}
+{#{% block extra_js %}{% endblock %}#}
```

Comment:
If there is a template which is overriding this block and mistakenly including some script twice, it should be fixed. Removing the block in the base template, so that *no* template's JS will be included and breaking lots of things, is not the solution.